testsuite/MDAnalysisTests/data/Amber/ache.prmtop

```diff
@@ -1,15 +1,15 @@
-%VERSION  VERSION_STAMP = V0001.000  DATE = 07/25/06  09:16:39                  
-%FLAG TITLE                                                                     
-%FORMAT(20a4)                                                                   
-NALA                                                                            
-%FLAG POINTERS                                                                  
-%FORMAT(10I8)                                                                   
+%VERSION  VERSION_STAMP = V0001.000  DATE = 07/13/23  13:12:57
+%FLAG TITLE
+%FORMAT(20a4)
+NALA
+%FLAG POINTERS
+%FORMAT(10I8)
```

Comment:
Does it matter that the new version does not have full white-space padded lines?

Reply:
No, it's a tell of which software was used to write the `.prmtop`. tleap uses fortran format writing, while parmed writes only the character that needs.